docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 api:
-  image: 'docker.io/c2corg/v6_api:latest'
+  image: 'docker.io/c2corg/v6_api:markdown-req'
```

Comment:
Do not forget to remove this when merging.

Reply:
I'm not sure to perfectly handle docker image generation process, but as far as I've understood, image docker.io/c2corg/v6_api:latest won't include good dependancies, so travis process will, so it won't build the docker image.

So I think we'll need at least one commit on master with `docker.io/c2corg/v6_api:markdown-req`, and then we could use `docker.io/c2corg/v6_api:master`

@mfournier, am I wrong ?

Reply:
Nope. Once your branch is merged into master, it will be latest, so it will be ok. You only need it in the meantime.

Reply:
Ok.